docs/_data/toc.yaml

```diff
@@ -46,7 +46,11 @@
 - title: REST API
   url: rest/rest-api
 - title: Thin Clients
-  url: thin-clients/index
+  items:
+    - title: Thin Clients Overview
```

Comment:
There is no thick/thin client distinction in Ignite 3, all clients are "thin".
The term "thin" can (and should) be omitted in documentation.